tests/e2e/test_load.py

```diff
@@ -0,0 +1,77 @@
+import os
+import random
+import string
+
+import pytest
+import torch
+import ssl
+import functools
+import certifi
+from azstoragetorch.io import BlobIO
+from azure.identity import DefaultAzureCredential
+from azure.storage.blob import BlobServiceClient
+
+
+@pytest.fixture(scope="module")
+def model():
+    ssl._create_default_https_context = functools.partial(
+        ssl.create_default_context, cafile=certifi.where()
+    )
+    model = torch.hub.load("pytorch/vision:v0.10.0", "resnet101", pretrained=False)
+    return model
+
+
+@pytest.fixture(scope="module")
+def upload_model(model, container_client):
+    blob_name = "model.pth"
+    torch.save(model.state_dict(), blob_name)
+    blob_client = container_client.get_blob_client(blob=blob_name)
+    with open(blob_name, "rb") as f:
+        blob_client.upload_blob(f)
+    return blob_name
+
+
+@pytest.fixture(scope="module")
+def account_url():
```

Comment:
Instead of writing the fixture again, let's move it out to a [`conftest.py`](https://docs.pytest.org/en/stable/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files) file so that we can share it between `test_load.py` and `test_read.py`.

Also, we will probably want to expand these shared fixtures to the `package` scope, instead of `module`, so that we use the same `account_url` and `container_client` for all end to end tests when we do this.